build.cake

```diff
@@ -47,16 +46,7 @@ Task("TestAndPackage")
     .IsDependentOn("Test")
     .IsDependentOn("Package");    
    
-Task("Clean")
-    .Does( ()=> 
-    {
-        CleanDirectory(artifactsDirectory);
```

Comment:
I couldn't get CleanDirectory to work reliably. Once artifactsDirectory had subdirectories CleanDirectory would fail.

Clean isn't really needed anyway, so I've just removed the task.